### DISCUSSION TITLE
My Arctic Code Vault Contributor badge disappeared

### DISCUSSION THREAD

**Discussion by nanofuxion** [community/community#20317] [Other Feature Feedback, Questions, & Ideas] [answered]
My Arctic Code Vault Contributor badge disappeared and I'm not quite sure why. 😔

**Comment by Tostisto** [comment]
Sometimes is useful re-add your email which is use in commit. In profile setting just try re-add your email.

**Comment by diegoleondev** [comment, marked as answer]
Hi @nanofuxion .
This happened to users who made contributions using the email address "noreply" provided by Github, but they replaced their actual user with "username" (example+username@users.noreply.github.com).

Github is currently investigating this problem, it is a matter of waiting for the report.

>**Note**
>Please **Mark as answer** if it was helpful.

**Comment by radar-blip** [comment]
There is an answer here that might help: 
https://github.community/t/lost-the-arctic-vault-badge-after-accidentally-deleting-email/176045

Here is another thread if you want to read through it:
https://github.community/t/arctic-vault-contributor-badge/194029
